bot.py

```diff
@@ -457,13 +458,35 @@ async def countdown(self, message, command_content):
         except ValueError:
             await message.channel.send(f":robot: {command_content} is not an integer.")
 
+    async def interrupt_play(self, message, source):
```

Comment:
What if you interrupt an interrupt_play?

Reply:
Depends on how you want it implemented... Right now only latest dinkster is resumed and original audio discarded.